examples/booklist.js

```diff
@@ -1,171 +1,133 @@
+/* jshint asi:true */
 /* Example!                                                    */
 /*   Grab a simple list of books from a random good reads user */
 /*                                                             */
 /* Note: Requires Goodreads Dev Keys (see below)               */
 
 /* Configuration                                               */
 /*   Get your keys at: http://www.goodreads.com/api/keys       */
-let key = process.env.GOODREADS_KEY;  //# Enter your key here to test!
-let secret = process.env.GOODREADS_SECRET; //# Enter your goodreads secret here to test!
+let key = process.env.GOODREADS_KEY  //# Enter your key here to test!
+let secret = process.env.GOODREADS_SECRET //# Enter your goodreads secret here to test!
 
 if (!key || !secret) {
-  console.log('You need to set your Goodreads dev Key and Secret!');
-  console.log('---');
-  console.log('1) Get them at:  http://www.goodreads.com/api/keys');
-  console.log('2) Set your key environment variable with: export GOODREADS_KEY=yourkey');
-  console.log('3) Set your secret environment variable with: export GOODREADS_SECRET=yoursecret');
-  console.log('---');
-  console.log('Having trouble? Ask me at @bdickason on Twitter.');
-  process.exit(1);
+  console.log('You need to set your Goodreads dev Key and Secret!')
+  console.log('---')
+  console.log('1) Get them at:  http://www.goodreads.com/api/keys')
+  console.log('2) Set your key environment variable with: export GOODREADS_KEY=yourkey')
```

Comment:
I'm not really a fan of going without semi-colons. If this can be reversed easily please.

Other wise thats fine. But, I'll want to add an eslint file soon.